src/utilities.js

```diff
@@ -52,4 +30,4 @@ export async function getItemClass(itemName, scene) {
 
 export async function setItemClass(key, value) {
     itemClasses[key] = value;
-}
\ No newline at end of file
+}
```

Comment:
why is there so much code SMH

Reply:
> why is there so much code SMH

It's a triple A title, what do you expect